lib/rules/selector-list-comma-newline-after/__tests__/index.js

```diff
@@ -130,50 +131,58 @@ testRule(rule, {
   reject: [
     {
       code: "a,b {}",
+      fixed: "a,\nb {}",
       message: messages.expectedAfter(),
       line: 1,
       column: 2
     },
     {
       code: "a, b {}",
+      fixed: "a,\nb {}",
       message: messages.expectedAfter(),
       line: 1,
       column: 2
     },
     {
       code: "a,  b {}",
+      fixed: "a,\nb {}",
       message: messages.expectedAfter(),
       line: 1,
       column: 2
     },
     {
       code: "a,\tb {}",
+      fixed: "a,\nb {}",
       message: messages.expectedAfter(),
       line: 1,
       column: 2
     },
     {
       code: "a,\nb,c {}",
+      fixed: "a,\nb,\nc {}",
       message: messages.expectedAfter(),
       line: 2,
       column: 2
     },
     {
       code: "a,\r\nb,c {}",
+      fixed: "a,\r\nb,\r\nc {}",
       description: "CRLF",
       message: messages.expectedAfter(),
       line: 2,
       column: 2
     },
     {
       code: "a, /* comment */ b {}",
+      fixed: "a,\n/* comment */ b {}",
```

Comment:
I'm not sure what we should do in this case. It really depends on comment content and how user uses it.

```css
div.hack, /* have to use hack to fix problem in IE6 */
.good {
}

.good,
/* button */.great {
}
```

Second example has HTML element in a comment for better understanding where selector applies.

In my experience second case is much more rare, than the first one. I think in this scenario it's better to fix like this:

```css
a, /* comment */\nb {}
```

Reply:
By reviewing `accept` tests I see, that `a, /* comment */\nb {}` is a correct fix, otherwise rule with autofix disabled will show a violation.